src/pages/Onboard/NameTeam.vue

```diff
@@ -130,18 +136,92 @@ export default {
       await this.createLicense()
       if (!this.updateServerError) this.goToResources()
     },
-    goToResources() {
+    async accept(pt) {
+      try {
+        this.accepting = true
```

Comment:
Should this be set to false in the `catch`?

Reply:
went back through and (hopefully) made the use of these more consistent.